README.md

```diff
@@ -1,2 +1,22 @@
 # doorhax
 Front-end for DubHacks 2019 project with @udomugo, @mkhsu, and @cforcomputer
+
+## DOORS
+(Directed Open Object Reality Simulator)
+
+## Inspiration
+We were inspired by a demo that we saw where someone created a model of a tardis that made it appear bigger on the inside using AR technology. We started to think about how we could apply this technique on a larger scale, and the potential community health benefits it could provide. 
```

Comment:
```suggestion
We were inspired by a demo that we saw where someone created a model of a TARDIS that made it appear bigger on the inside using AR technology. We started to think about how we could apply this technique on a larger scale, and the potential community health benefits it could provide. 
```